tests/end_to_end/UniswapV2/test_uniswap_v2_router.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+
+@pytest.mark.asyncio(scope="session")
+# @pytest.mark.UniswapV2Router
+class TestUniswapV2Router:
+    class TestDeploy:
+        async def test_should_set_constants(self, router, weth, factory):
+            assert await router.WETH() == weth.address
+            print("WETH Address", weth.address)
```

Comment:
if you print, I think that you don't know how to use a debugger
1. select python interpreter
2. configure test
3. debug test